apis/fluent/atrium-api-fluent/src/commonMain/kotlin/ch/tutteli/atrium/api/fluent/en_GB/iterableFeatureExtractors.kt

```diff
@@ -59,3 +61,29 @@ fun <E : Comparable<E>, T : Iterable<E>> Expect<T>.max(): Expect<E> =
  */
 fun <E : Comparable<E>, T : Iterable<E>> Expect<T>.max(assertionCreator: Expect<E>.() -> Unit): Expect<T> =
     _logic.max(::identity).collectAndAppend(assertionCreator)
+
+/**
+ * Expects that Iterable is not empty and returns an [Expect] for the last element in it.
+ *
+ * @return The newly created [Expect] for last element.
+ *
+ * @sample ch.tutteli.atrium.api.fluent.en_GB.samples.ListFeatureExtractorSamples.lastFeature
+ *
+ * @since 1.2.0
+ */
+val <E, T: Iterable<E>> Expect<T>.last: Expect<E?>
```

Comment:
no idea why you started adding `?` here and there but the implementation you had in List was just fine :)
```suggestion
val <E, T: Iterable<E>> Expect<T>.last: Expect<E>
```